src/work-allocation/services/work-allocation-case.service.ts

```diff
@@ -47,7 +47,7 @@ export class WorkAllocationCaseService {
   }
 
   public getTasksByCaseId(caseId: string): Observable<Task[]> {
-    return this.http.post<Task[]>(`${BASE_URL}/task/${caseId}`, {refined: false});
+    return this.http.post<Task[]>(`${BASE_URL}/task/${caseId}`, {refined: true});
```

Comment:
This will defeat the purpose of the refined field, basically meaning new permission logic is used every time (see below for what we used to use for this). Is this what we want?

Reply:
Yes please, thats fine, If you see the discussion. From now on we will always enable it as true